app/src/components/ConvictionVisuals.js

```diff
@@ -189,15 +207,31 @@ export function ConvictionCountdown({ proposal }) {
       <Text color={theme.negative.toString()}> ✘ Won't pass</Text>
       <div>
         <Text color={theme.surfaceContent.toString()}>
-          Insufficient staked tokens
+          {!isNaN(neededTokens)
+            ? 'Insufficient staked tokens'
+            : 'Not enough funds in the organization'}
         </Text>
         <br />
         <Text color={theme.surfaceContentSecondary.toString()}>
-          (At least{' '}
-          <Tag>
-            {neededTokens} {tokenSymbol}
-          </Tag>{' '}
-          more needed).
+          (
+          {!isNaN(neededTokens) ? (
+            <React.Fragment>
+              At least{' '}
+              <Tag>
+                {parseFloat(
+                  fromDecimals(neededTokens.toString(), tokenDecimals)
+                )
+                  .toFixed(2)
+                  .toString()}{' '}
+                {tokenSymbol}
+              </Tag>{' '}
+              more needed
+            </React.Fragment>
+          ) : (
+            `Funding requests must be below ${defaultBeta *
```

Comment:
You can obtain `beta` value from `appState.generalParams.maxRatio`. The `defaultBeta` constant doesn't have to be exposed anymore.